packages/web/src/components/ui/HeaderNew/HeaderNew.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import styles from './HeaderNew.module.scss';
+import Container from '../Container/Container';
+import { Github, Logo } from 'components/icons';
+import { trackCustomEvent } from '../../../services/analytics';
+
+export default function HeaderNew() {
+  return (
+    <Container>
+      <header className={styles.header}>
+        <Logo className={styles.logo} />
```

Comment:
I think we should not pass external class names into components. If we need to change component representation, we better define some variants and pass constant here

Reply:
Simply adding a wrapper would also suffice, if I'm not mistaken.